environments/production/inventory.ini

```diff
@@ -188,6 +188,32 @@ hqes7.internal-va.commcarehq.org elasticsearch_node_name=hqes7
 [hqesmaster0]
 172.24.32.67 hostname=hqesmaster0 elasticsearch_node_name=hqesmaster0
 
+[es0]
+10.202.41.22 hostname=es0-production elasticsearch_node_name=es0 ec2=yes
+
+[es1]
+10.202.41.96 hostname=es1-production elasticsearch_node_name=es1 ec2=yes
```

Comment:
I think we'll want to set the `elasticsearch_node_zone` on all of the ES nodes (new and old) before deploying this:

https://github.com/dimagi/commcare-cloud/blob/7809a83a88ff77461aad3bbcba851409b2922733/src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2#L10-L14

Reply:
Yup, I agree. That's the next step that I wanted to discuss in more detail with you this afternoon

Reply:
But I can add that piece of it now

Reply:
Oh, ha. Jokes. That's already in the PR under `[es_data_aws:vars]`

Reply:
hah, oops, missed that  :mag: